helper_test.go

```diff
@@ -894,6 +894,32 @@ func createOrganizationToken(t *testing.T, client *Client, org *Organization) (*
 	}
 }
 
+func createOrganizationTokenWithOptions(t *testing.T, client *Client, org *Organization) (*OrganizationToken, func()) {
```

Comment:
I would expect that this would take an `OrganizationTokenCreateOptions` parameter, and pass it to the client [here](https://github.com/hashicorp/go-tfe/pull/672/files#diff-9c616bd91858daac3bdd7b717814be037a98b9f9ae75134b6e3815fa9a6c231fR905)